### DISCUSSION TITLE
How do I contact the standing committee now?

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#91] [Team Posts]
Being an admin, I can see and post to the SC team but that actually seems more like an unfortunate accident (I simply can access that because I'm an admin; I shouldn't really be looking in there, and I'm not :) ).

I think the mailing lists are down now? What do we do now if we want to email the SC?..

**Comment by emilymbender** [comment]
For the moment, even the SC is just listing out individual emails when we try to contact each other. I'd love a more robust solution.

**Comment by olzama** [comment]
I see. A more robust solution *may* be consolidating admins with the SC (I believe we have discussed this before). This basically would mean that the SC must at all times include at least a couple members who are committed to actively administering github.

**Comment by olzama** [comment]
Actually, I suppose that won't solve the problem. It solves only the problem of non-SC members who are admins being able to see the SC content. It doesn't solve the problem of contacting SC at all, my bad.

**Comment by guyemerson** [comment]
Replying belatedly (I've now returned from leave)...

It seems GitHub doesn't have any functionality for sending messages privately.  So it's possible to contact the standing committee by starting a discussion and tagging the standing committee (which will send notifications), but this would be visible to everyone in Delph-in (they won't get notified but they can still browse discussions and read it).  So if we want to let people send private messages to the standing committee (other than listing out emails), then we would need to set up a separate mailing list.  Google Groups would be one option, for example.

**Comment by arademaker** [comment]
Hi all, @olzama made us aware of https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/. I was expecting a transparent migration seems no warning about messages visibility and access rights were made in the link. Since the only option we have is to migrate or lose the control of the messages (freeze them as ready-only), I pushed the button moving the https://github.com/orgs/delph-in/teams/participants and https://github.com/orgs/delph-in/teams/admins to https://github.com/orgs/delph-in/discussions/categories/team-posts. My fault! The docs repository is the one that hosts the organization level discussions: https://github.com/delph-in/docs/discussions. Repositories can still have separated discussion forums.

It seems now that both the https://github.com/orgs/delph-in/teams/participants and https://github.com/orgs/delph-in/teams/admins forums were collapsed in the https://github.com/orgs/delph-in/discussions/categories/team-posts. Not sure if we still have any way to create forums restricted to specific teams, as @guyemerson said above, maybe not.

Is there any message we must delete maybe?

**Comment by arademaker** [comment]
Indeed, they did say something about the visibility... but I didn't pay attention! 

<img width="461" alt="image" src="https://user-images.githubusercontent.com/86403/219740770-6dc925f4-374a-4ff8-844a-4724cd6dab56.png">
